File: Lab5/CS/Letters.cs
using System;
using System.Collections.Generic;
using System.Security.Cryptography.X509Certificates;
using System.Text;

namespace Lab5
{
    internal class Letters : Lines
    {
        private string mainStr;
        public string str;

        public Letters() 
        {
            
        }

        public Letters(string ret_string) 
        {
            str = ret_string;
            mainStr = str;
        }

        public override int Length()
        {
            return str.Length;
        }

        public override string Process()
        {
            char[] arr = str.ToCharArray();
            Array.Reverse(arr);
            return new string(arr);
        }

    }
}

File: Lab5/CPP/Letters.cpp
#include <iostream>
#include <string>
#include <algorithm>
#include "Letters.h"


int Letters::GetLength() 
{
	return str.length();
}

std::string Letters::Process() 
{
	std::string ret = str;
	std::reverse(ret.begin(), ret.end());
	return ret;
}
